src/components/cardCustom/featureCard/FeatureCard.tsx

```diff
@@ -1,18 +1,14 @@
 import React from 'react';
 import { Card, Group, Text } from '@mantine/core';
-import './DestinationCard.scss';
+import './FeatureCard.scss';
 
-interface DestinationCardProps {
+interface FeatureCardProps {
   icon: React.ReactNode;
   title: string;
   description: string;
 }
 
-export function DestinationCard({
-  icon,
-  title,
-  description,
-}: DestinationCardProps) {
+export function FeatureCard({ icon, title, description }: FeatureCardProps) {
```

Comment:
[No function, only arrow func](https://www.w3schools.com/js/js_arrow_function.asp)

```suggestion
export const FeatureCard = ({ icon, title, description }: FeatureCardProps)  => {
```